src/library/autodj/dlgautodj.cpp

```diff
@@ -83,6 +83,21 @@ DlgAutoDJ::DlgAutoDJ(QWidget* parent,
     autoDJStateChanged(m_pAutoDJProcessor->getState());
 
     updateSelectionInfo();
+
+    // populate the mode combobox
+    auto selectors = m_pAutoDJProcessor->getSelectorList();
+    foreach(TrackSelector* selector, selectors)
+    {
+        comboBoxSelector->addItem(selector->getName(), selector->getId());
+        if(m_pAutoDJProcessor->getSelector() &&
```

Comment:
'if' and other keywords are neither functions nor macros. It is common practice to place a space between the keyword and the opening brace.